setup/blocks/base_block.php

```diff
@@ -1,28 +1,53 @@
 <?php
 
-class BaseBlock {
-    public function __construct($block_name = '', $block_params = array(), $group_params = array(), $fields_params = array()){
-
-        if (empty($block_name) || empty($block_params) || empty($group_params) || empty($fields_params)) {
```

Comment:
Questo rende i parametri obbligatori per funzionare per cui non mi era chiaro perché avessi passato dei default nella firma del metodo. In ogni caso ho cambiato radicalmente l'approccio quindi il problema non sussiste.